extensions/amp-smartlinks/validator-amp-smartlinks.protoascii

```diff
@@ -0,0 +1,53 @@
+#
+# Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-smartlinks
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-smartlinks"
+    version: "0.1"
+    version: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+
+tags: {  # <amp-smartlinks>
+  html_format: AMP
+  tag_name: "AMP-SMARTLINKS"
+  requires_extension: "amp-smartlinks"
+  attrs: {
```

Comment:
Are there limits on what the values should be for these attributes?

For instance in the markdown `linkmate` is set to `true`.

Is it just `true` or `false`? If so then it can limit the value by adding:
```
  value: "false"
  value: "true"
```

Or is it merely it's presence that determines `true` or `false`? If so it doesn't need any value and can be enforced with:
```
  value: ""
```

Consider what limits make sense for the other attributes here.

Reply:
To follow up, if it's only true and false, then perhaps merely it's presence can be the indicator then. If attribute `exclusive-links` is present then that means true, otherwise it'll be interpreted as false. To do this then it'd be `value: ""`.